index.js

```diff
@@ -100,23 +129,29 @@ function myMap(array, callback) {
 // ***** Callbacks - Question 1 *****
 
 // *** Uncomment the lines below to test
-// console.log("%cCallbacks - Question 1", "color: red")
+console.log("%cCallbacks - Question 1", "color: red")
 
-// console.log(myMap([1,2,3,4,5], triple)) 
-// // => [3,6,9,12,15]
+function triple(value, i, arr) {
+  return value * 3;
+}
```

Comment:
This works however, you are calling the default arguments for forEach (value, i, arr). function triple does not have any default arguments. the "i" and "arr" you added are essentially placeholders for extra arguments you could pass to triple when called.  Instead, you can just do 
function triple(value) {
   return value * 3
}